00-boilerplate/readme_es.md

```diff
@@ -0,0 +1,20 @@
+# 02 Web Boiler plate
+
+## Resumen
```

Comment:
Just add the right text to this readme so anybody can understand what is this example for

Reply:
done